src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -478,11 +478,10 @@ export class MeetingManager implements AudioVideoObserver {
       this.selectedAudioInputDevice = device;
       this.publishSelectedAudioInputDevice();
     } catch (error) {
-      console.error(
-        'MeetingManager failed to select audio input device',
-        error
-      );
-      throw new Error('MeetingManager failed to select audio input device');
+      error.message =
```

Comment:
Do we need to type check `error` before assigning `error.message`? If all error that are caught here are instance of `Error` then `error.message` is fine otherwise can throw runtime error.

Reply:
From JS SDK this `error` is 100% an instance of `Error`. But React SDK does not have this info so it's good to do a type check here. Good call! 

Strange that typescript does not give me an error for this.